frontend/src/features/recommendation/LanguageFilter.tsx

```diff
@@ -10,11 +11,16 @@ interface Props {
 function LanguageFilter(props: Props) {
   const { t } = useTranslation();
 
-  const languageChoices = {
-    en: t('language.english'),
-    fr: t('language.french'),
-    de: t('language.german'),
-  };
+  const languageChoices: Record<string, string> = useMemo(
+    () =>
+      Object.fromEntries(
+        availableRecommendationsLanguages.map((language) => [
+          language,
+          t(`language.${language}`),
```

Comment:
Unfortunately, i18next-parser is not able to understand that 3 different translations keys are used here. When running `yarn run i18n:parse` the existing translation keys will be deleted.

I think that we need to keep an explicit object with all translation keys, similarly to how the criteria names are currently translated:
https://github.com/tournesol-app/tournesol/blob/0b6718530ea7db21ee5f9a9fcc5d2be8b156c829/frontend/src/utils/constants.ts#L30-L44

Reply:
Ok, it should be fixed now.